R/importSNV.TCGA.R

```diff
@@ -0,0 +1,323 @@
+                        "3'UTR", "3'Flank", "5'UTR", "5'Flank", "IGR1" , "Intron", "RNA", "Targeted_Region")
+            myVclasses <- unique(var_class_)
+            if (! all(myVclasses %in% classVknown)) {
+                # drop file
+                classValid <- FALSE
+            } else {
+                classValid <- TRUE
+            }
+
+            # check if vairant_type contains valid values from known list
+            typeVknown <- c("SNP", "DNP", "TNP", "ONP", "INS", "DEL", "Consolidated2")
+            myVtypes <- unique(var_type_)
+            if (! all(myVtypes %in% typeVknown)) {
+                # drop file
+                typeValid <- FALSE
+            } else {
+                typeValid <- TRUE
+            }
+
+            # check chromosome value is valid
+            typeChr <- c("chr1","chr2","chr3","chr4","chr5","chr6","chr7","chr8","chr9","chr10","chr11","chr12","chr13",
+                         "chr14","chr15","chr16","chr17","chr18","chr19","chr20","chr21","chr22","chrX","chrY","chrM")
+            myVchr <- unique(chr_)
+            if (! all(myVchr %in% typeChr)) {
+                # drop file
+                chrValid <- FALSE
+            } else {
+                chrValid <- TRUE
+            }
+
+            if (classValid & typeValid & chrValid == FALSE) {
+                # there was an invalid value
+                # report error in file and log
+                # drop file
+                # go to next
+                myNotes <- c(paste(i, " - File was dropped due to invalid attributes in column chr/class/type"))
+                numFilesInvalid <- numFilesInvalid + 1
+            } else {
+                # all tests passed
+                # add data to rSNV
+                fileasDT <- cbind(hugo_ , chr_ , start_, end_, strand_, var_class_,
+                                  var_type_, ref_allele_, tum_all1_, tum_all2_,
+                                  tum_samp_barcode_, tum_sam_uuid_)
+                if (na.rm) {
+                    na.omit(fileasDT)
+                }
+                write.table(fileasDT,  file=rSNV, sep="\t", append=TRUE, row.names =
+                                FALSE, col.names = FALSE, quote = FALSE)
+            }
+            filesFinished <- filesFinished + 1
+        }
+
+
+        # ==== WRITE LOG ===========================================================
+
+        if(writeLog) {
+
+            myTitle <- "importSNV.TCGA"
+
+            # Compile function call record
+            myCall <- character()
+            myCall[1] <- "importSNV.TCGA("
+            myCall[2] <- sprintf("fMAF = \"%s\", ", paste(fMAF, collapse=" "))
+            myCall[3] <- sprintf("rSNV = \"%s\", ", rSNV)
+            myCall[3] <- sprintf("silent = %s, ", as.character(silent))
+            myCall[4] <- sprintf("writeLog = %s)", as.character(writeLog))
+            myCall[5] <- sprintf("na.rm = %s)", as.character(na.rm))
+            myCall <- paste0(myCall, collapse = "")
```

Comment:
Missing the closing bracket of the call. Arguments not separated. Did you try this?